apps/test/unit/templates/rubrics/LearningGoalTest.jsx

```diff
@@ -129,4 +129,38 @@ describe('LearningGoal', () => {
     );
     sendEventSpy.restore();
   });
+
+  it('displays Evaluate when AI is disabled and no understanding has been selected', () => {
+    const wrapper = mount(
+      <LearningGoal
+        learningGoal={{
+          learningGoal: 'Testing',
+          aiEnabled: false,
+          evidenceLevels: [],
+        }}
+        teacherHasEnabledAi
+        canProvideFeedback
+      />
+    );
+    wrapper.update();
+    expect(wrapper.find('BodyThreeText').text()).to.include('Evaluate');
+    wrapper.unmount();
```

Comment:
out of curiosity, why is this needed?

Reply:
I noticed in the enzyme documentation that you should always unmount after using the mount command to prevent memory leaks.

Reply:
huh TIL!

Reply:
yes, nice find!